src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
-server.port = 8082
\ No newline at end of file
+server.port = 8082
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/testdb
+spring.datasource.username= postgres
+spring.datasource.password= rodrigo
+
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+
+spring.jpa.hibernate.ddl-auto = update
```

Comment:
We should leave this as 'validate'. Otherwise, we could be triggering a modification to the database that we don't want to apply (yet). So it's a good practice to version this file with a 'validate' in it instead of an update.